tests/cli/test_job.py

```diff
@@ -96,3 +97,33 @@ def test_calc_statuses__check_defaults__all_statuses_true(
     assert not std.out
     assert not std.err
     assert not caplog.text
+
+
+@pytest.mark.parametrize(
+    "env_var", ["NP_STORAGE_ROOT_MOUNTPOINT", "NP_STORAGE_ROOT_MOUNTPOINT=value"]
+)
+def test_build_env_reserved_env_var_conflict_passed_as_parameter(env_var: str) -> None:
```

Comment:
apart from these two tests, `build_env` is not covered

Reply:
should we properly cover it then?

Reply:
sure. but not in this PR. Issue: https://github.com/neuromation/platform-client-python/issues/995